ee/tabby-schema/src/schema/mod.rs

```diff
@@ -834,6 +834,7 @@ impl Mutation {
         password1: String,
         password2: String,
         invitation_code: Option<String>,
+        name: Option<String>
```

Comment:
```suggestion
        name: String
```
You can put `Some(name)` in line 848.